Update home about

ash-arl opened the pull request:
Added required section to the home page and worked on the UI.

sovesa commented:
Can you please add a screenshot of section you added?
And explain which UI you have worked on?

ash-arl commented:
> Can you please add a screenshot of section you added? And explain which UI you have worked on?

Added On Going Course and Upcoming Course
Added Upcoming Events 
Added Volunteer Section which works differently if the user is logged in or not
Added Testimonials
Added scroll animations to the elements in the home page
